Firmware/RTK_Surveyor/Begin.ino

```diff
@@ -29,14 +27,89 @@ void beginBoard()
   {
     productVariant = REFERENCE_STATION;
   }
-  else if (isConnected(0x19) == true) //Check for accelerometer
```

Comment:
Please do not remove this check. Early versions of the RTK Express had only the Accel added, and no ID resistors. Therefore, this line allows backwards compatibility.

Reply:
Never mind. I see your check below.